lib/pdk/Http.v3

```diff
@@ -0,0 +1,57 @@
+// `Request` represents an HTTP request made by the Extism host.
+class Request {
```

Comment:
If you like you can shorten this up with:

```
class Request(var method: Method, var header: Header, var url: string) { ...
```

If they're intended to be immutable, then you can drop the `var`.